docs/symfony/get-it-started.md

```diff
@@ -135,7 +166,7 @@ class PaymentController extends Controller
         $captureToken = $this->get('payum')->getTokenFactory()->createCaptureToken(
             $gatewayName, 
             $payment, 
-            'done' // the route to redirect after capture
```

Comment:
nope it should be `done`.

Reply:
Hm, this is what my `\vendor\payum\payum-bundle\Resources\config\routing\capture.xml` looks like (Symfony):

```xml
<?xml version="1.0" encoding="UTF-8" ?>

<routes xmlns="http://symfony.com/schema/routing" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/routing http://symfony.com/schema/routing/routing-1.0.xsd">
    <route id="payum_capture_do_session" path="/payment/capture/session-token">
        <default key="_controller">PayumBundle:Capture:doSessionToken</default>
    </route>

    <route id="payum_capture_do" path="/payment/capture/{payum_token}">
        <default key="_controller">PayumBundle:Capture:do</default>
    </route>
</routes>
```

I have no route `done` there. And I got an error when I tried it with `done`, so I figured I have to change it to `payum_capture_do`

Reply:
`done` route is one created by you in your app. It is not part of payum bundle.

Reply:
Uh, OK! Then this would be the right place to explain that :-)

* What is the route/controller supposed to do? (In which situation is it called?)
* Any parameters?

Reply:
> What is the route/controller supposed to do? (In which situation is it called?)

The payment process should always end up there, regardless of payment status. Inside it, you have to check the payment status [like this] and do whatever your business dictates you to do. 
 (https://github.com/makasim/PayumBundleSandbox/blob/23200c1aab156eef0f7c1e3f2373ab7414dce27e/src/Acme/PaymentBundle/Controller/DetailsController.php#L27).

Reply:
> Any parameters?

Nope, everything you need is bound to the token already  (a payment and its details)

Reply:
So the `done` route is where the user is being redirected to, when returning from the (e.g.) PayPal website? So that's what most payment providers call the "success url"?

Reply:
Sorry, what you're saying can't be right ;-) The purpose of this `prepareAction` is to set up the payment ("capture"), and then redirect the user to the payment provider, right? If I enter one of my own routes as this argument, the user gets redirected there. This doesn't make any sense, or does it?

Reply:
> So the done route is where the user is being redirected to, when returning from the (e.g.) PayPal website? So that's what most payment providers call the "success url"?

nope, the user comes from Paypal back to capture url and only after it is redirect to done url (by capture action)

Reply:
here a user is being redirected to done url https://github.com/Payum/PayumBundle/blob/master/Controller/CaptureController.php#L39